packages/teams-js/src/public/meeting.ts

```diff
@@ -925,6 +925,173 @@ export namespace meeting {
     );
   }
 
+  /**
+   * This function is used to join a private meeting.
+   * This opens a meeting in a new window for the desktop app.
+   * In case of a web app, it will close the current app and open the meeting in the same tab.
+   * @param joinPrivateMeetingParams The parameters for joining the private meeting. {@link JoinPrivateMeetingParams}
+   * @throws error if the meeting join fails, the promise will reject to an object with the error message.
+   */
+  export function joinPrivateMeeting(joinPrivateMeetingParams: JoinPrivateMeetingParams): Promise<void> {
+    if (!validateJoinMeetingParamsBasic(joinPrivateMeetingParams)) {
+      return Promise.reject(new Error('Invalid joinPrivateMeetingParams'));
+    }
+
+    joinPrivateMeetingParams.joinWebUrl = new URL(joinPrivateMeetingParams.joinWebUrl);
+
+    const serializedJoinPrivateMeetingParams = {
+      joinWebUrl: joinPrivateMeetingParams.joinWebUrl.href,
+      subject: joinPrivateMeetingParams.subject,
+      source: joinPrivateMeetingParams.source || EventActionSource.Other,
+      threadId: joinPrivateMeetingParams.threadId,
+      messageId: '0',
```

Comment:
Could you put a comment inside of this function explaining why it's okay to set `messageId` and `replyChainMessageId` to 0? (It's because those values aren't used when joining private meetings, and I assume 0 is sort of a default "ignore this value" value in the Teams backend?)

Reply:
Removed subject, messageId, replyChainMessageId and threadId after consulting CMD team.